lib/rubocop/cop/style/command_literal.rb

```diff
@@ -0,0 +1,98 @@
+# encoding: utf-8
+
+module RuboCop
+  module Cop
+    module Style
+      # This cop enforces using `` or %x around command literals.
+      class CommandLiteral < Cop
+        include ConfigurableEnforcedStyle
+
+        MSG_USE_BACKTICKS = 'Prefer using backticks around command string.'
```

Comment:
prefer using -> use

One of those days we should make all the messages uniform.